pkg/apis/machine/v1beta1/machine_webhook.go

```diff
@@ -113,6 +110,10 @@ const (
 	defaultGCPCredentialsSecret = "gcp-cloud-credentials"
 	defaultGCPDiskSizeGb        = 128
 	defaultGCPDiskType          = "pd-standard"
+	// https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.6/46.82.202007212240-0/x86_64/meta.json
+	// https://github.com/openshift/installer/pull/3808
+	// https://github.com/openshift/installer/blob/d75bf7ad98124b901ae7e22b5595e0392ed6ea3c/data/data/rhcos.json
+	defaultGCPDiskImage = "projects/rhcos-cloud/global/images/rhcos-46-82-202007212240-0-gcp-x86-64"
```

Comment:
Do we expect this to change often?

Reply:
I don't believe so. We could choose to change it one per release. This is the instance bootimage, the early pivot will take over and get you to the latest os version after bootstrapping a new instance.
If/when MCO makes the image cluster version available we could dynamically default to it.